shared/common-adapters/emoji.desktop.js

```diff
@@ -10,9 +10,22 @@ import type {Props} from './emoji'
 
 const backgroundImageFn = (set: string, sheetSize: string) => emojiSet
 
+const skinToneRegex = /:([^:]*):(:skin-tone-\d:)?/i
+
+const isValidEmoji = (fullEmoji: string) => {
+  const parts = skinToneRegex.exec(fullEmoji)
+  if (parts) {
+    return !parts[1].startsWith('skin-tone') && emojiIndex.search(parts[1]).length > 0
```

Comment:
main problem was when an emoji like `:skin-tone-X::something:` was being passed in. also doesn't seem to affect mobile at all.